mutation-testing/check-packages-and-shards/action.yml

```diff
@@ -0,0 +1,123 @@
+name: Check Packages and Shards
+
+outputs:
+  run_big_packages:
+    value: ${{ steps.check_packages_and_shards.outputs.run_big_packages }}
+  big_packages_with_shards:
+    value: ${{ steps.check_packages_and_shards.outputs.big_packages_with_shards }}
+  run_small_packages:
+    value: ${{ steps.check_packages_and_shards.outputs.run_small_packages }}
+  small_packages_with_shards:
+    value: ${{ steps.check_packages_and_shards.outputs.small_packages_with_shards }}
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Checkout stacks-core repo
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install cargo-mutants
+      shell: bash
+      run: cargo install --version 23.12.2 cargo-mutants
+
+    - name: Relative diff
+      shell: bash
+      run: |
+        git diff origin/${{ github.base_ref }}.. > git.diff
+
+    - name: Update git diff
+      shell: bash
+      run: |
+        input_file="git.diff"
+        temp_file="temp_diff_file.diff"
+
+        # Check if the commands exist on the host
+        for cmd in tac awk sed; do
+          command -v "${cmd}" > /dev/null 2>&1 || echo "Missing command: ${cmd}"
```

Comment:
there should be an exit code here if a required binary is missing. 
an example i've used looks like this:
https://gist.github.com/wileyj/9922af657ae8048cb67ba9621141e7e1#file-seed-sh-L61-L68
https://gist.github.com/wileyj/9922af657ae8048cb67ba9621141e7e1#file-seed-sh-L80-L82

basically, a small function that prints a message and exits with non-zero code